package.json

```diff
@@ -20,7 +20,7 @@
     "check": "yarn run lint && yarn test",
     "rules-status": "eslint-index lib/config/all.js --format table",
     "rules-omitted": "eslint-index lib/config/all.js --status omitted",
-    "lint": "eslint . --max-warnings 0 --format codeframe",
+    "lint": "eslint . --max-warnings 0 --format codeframe --ignore-pattern tests/fixtures",
```

Comment:
I had to change this to prevent the repo lint from complaining about intentionally bad files in fixtures that I need to still result in errors when we lint from a test.

Reply:
Can you move this into the .eslintignore file: https://github.com/Shopify/eslint-plugin-shopify/blob/master/.eslintignore

Reply:
I'm not sure if I can use the global ignores because it will affect the tests that are validating the files we want CI `lint` to ignore but not CI `test`

Reply:
new plan, --no-ignore can override the global ignores for these tests